CONTRIBUTING.md

```diff
@@ -29,12 +29,16 @@ npm test
 
 ### Publish
 
-To publish a new version is a little complecated. We need to prepend the right version metadata to each generated dist file before adding a git tag and publishing to npm.
+First please install lerna globally.
 
 ```sh
-# First we need to install lerna globally
 npm install --global lerna
-# Next let's update all packages's version without execute any git or npm command
+```
+
+To publish a new version is a little complecated. We need to prepend the version metadata to each generated dist file before adding a git tag and publishing to npm.

```

Comment:
typo: complicated.

Reply:
The `npm test` is actually stylelint..... what about change this task's name to: lint

Reply:
`npm run bootstrap` will run lerna, but `npm install --global lerna` is at bottom of the page. It might cause errors.

Reply:
Fixed typo, thanks.
I renamed `test` to `lint` and add a `npm test` script which runs `npm run lint`

> `npm run bootstrap` will run lerna, but `npm install --global lerna` is at bottom of the page. It might cause errors.

This is by design. `npm run bootstrap` will run `lerna` from the `node_modules/.bin/lerna`, without install `lerna` globally.

Most developers should never run commands like `lerna publish`, for them the npm scripts `npm run build` is enough to use.

Reply:
ok